src/tutorials/code/node/constellation/2.js

```diff
@@ -0,0 +1 @@
+const ca = new Carina({ isBot: true }).open();
```

Comment:
Also, this should be the default... our frontend should be the **only** consumer of Carina which would not advertise that it's a bot